src/components/LoginPage/LoginPageSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const slice = createSlice({
+  name: 'login',
+  initialState: {
+    gapiInitialized: false,
+  },
+  reducers: {
+    initGapi: state => {
+      state.gapiInitialized = true;
```

Comment:
Gapi is not only for login. It's for the whole app.
Why did you move it to `LoginPageSlice` ?